pkgs/by-name/in/insync-nautilus/package.nix

```diff
@@ -1,47 +1,42 @@
 {
+  lib,
   stdenv,
   fetchurl,
-  lib,
   dpkg,
   nautilus-python,
-  insync-emblem-icons,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "insync-nautilus";
-  version = lib.getVersion insync-emblem-icons;
```

Comment:
okay, that's fine. so, I guess we couldn't find latest insync-emblem-icon anywhere.